index.html

```diff
@@ -12,40 +12,171 @@
 
 </head>
 <body>
-    <header class="toolbar">
-        <nav>
-            <a><img src="images/LogoPlaceholder.png" alt="My Logo"/></a>
-            <img class="menu" src="images/hamburger.png" alt="menu"/>
-        </nav>
-    </header>
-
-    <section class="headline">
-        <div class="heading"><h1>I’m Cathy<br>
-            Glad to see<br> you!</h1>
-        </div>
-        <div class="paragraph"><p>I’m a software developer! I can help you build a product , feature or website.Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        </div>
-        <div class="connect">
-            <p>LET’S CONNECT</p>
-        </div>
-        <br>
-        <ul class="contact">
-            <li class="twit">
-                <a ><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a>
-            </li>
-            <li>
-                <a ><img class="linkedin" src="images/Linkedinicon@2x.png" alt="linkedin"></a>
-            </li>
-            <li>
-                <a ><img class="email" src="images/Mail@2x.png" alt="email"></a>
-            </li>
-            <li>
-                <a ><img class="github" src="images/Github@2x.png" alt="Github"></a>
-            </li>
-            <li>
-                <a ><img class="vector" src="images/Vector@2x.png" alt="Vector"></a>
-            </li>
-        </ul>
+    <section class="main">
+        <section class="home">
+            <header class="toolbar">
+                <nav>
+                    <a><img src="images/LogoPlaceholder.png" alt="My Logo"/></a>
+                    <img class="menu" src="images/hamburger.png" alt="menu"/>
+                </nav>
+            </header>
```

Comment:
Please put this part as a child for the body as you did previously because it's more semantic